modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -0,0 +1,148 @@
+** A `ProjectID` must be 6 to 30 characters in length, can only contain lowercase letters, numbers, and hyphens, and must start with a letter, and cannot end with a hyphen.
+* A tag key and value must have a minimum of 1 character and can have a maximum of 63 characters.
+* A tag key must contain only uppercase and lowercase alphanumeric characters, hyphens, underscores, and periods (that is, -_.).
+* A tag value must contain only uppercase and lowercase alphanumeric characters, hyphens, underscores, periods, at signs, percent signs, equals signs, plusses, colons, commas, asterisks, pound signs, ampersands, parentheses, square braces, curly braces, and spaces (that is, -_.@%=+:,*$&()[]{} and spaces).
+* A tag key and value must begin and end with an alphanumeric character.
+* Tag value is of type scalar and must be one of the pre-defined values for the key.
+* You can configure a maximum of 50 tags.
+* There should be no tag key with the same value as any of the existing tag keys on a resource (inherited from parent resource).
+
+.Post-cluster creation
+
+After creating {product-title} cluster, you can access the list of the labels and tags defined for the GCP resources by referring to the new fields `.status.platformStatus.gcp.resourceLabels` and `.status.platformStatus.gcp.resourceTags` present in the `infrastructures.config.openshift.io/cluster` object.
+
+.Sample `infrastructure` file
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Infrastructure
+metadata:
+ name: cluster
+spec:
+ platformSpec:
+   type: GCP
+status:
+ infrastructureName: <cluster_id>
+ platform: GCP
+ platformStatus:
+   gcp:
+     resourceLabels:
+     - key: <label_key>
+       value: <label_value>
+     resourceTags:
+     - key: <tag_key_short_name>
+       parentID: <OrganizationID/ProjectID>
+       value: <tag_value_short_name>
+   type: GCP
+----
+
+Along with the user-defined labels, resources will also have a label defined by the {product-title}. The format of the {product-title} labels is `kubernetes-io-cluster-<cluster_id>:owned`. The `cluster_id` parameter is the value of `.status.infrastructureName` present in `infrastructures.config.openshift.io/cluster`.
+
+User-defined and {product-title} labels can be found on the following GCP resoucres:
+
+* Compute Instance
+* Compute Disk
+* Compute Forwarding Rule
+* Compute Image
+* DNS Managed Zone
+* Filestore Instance
+* Storage Bucket
+
+User-defined tags can be found added to the GCP resources, that is, storage bucket.
+
+The following are the limitations for the user-defined tags:
+
+* Tags will not be attached to master compute instances and storage bucket created by `installer`, worker compute instances created by `machine-api-provider-gcp controller`, filestore instance resources created by `gcp-filestore-csi-driver-operator` and compute disk, compute image resources created by `gcp-pd-csi-driver-operator`.
```

Comment:
- Remove future tense
- what is 'master compute' instances here? We generally don't use master term but depends on the context.same applies to worker.
- What is installer? If it is not an object type, we have to simply call it installer without quotes.

Reply:
installer > installation program

Reply:
Ack.

Reply:
Hi @subhtk . I'd say you can ignore this. Apologies. I see that the [Machine Management ](https://docs.openshift.com/container-platform/4.13/machine_management/creating-infrastructure-machinesets.html#binding-infra-node-workloads-using-taints-tolerations_creating-infrastructure-machinesets) docs  still use the value, so the engineering codebase must not be updating the name of this node.

Reply:
After you resolve the merge conflict, this PR is ready for merging. Good work!

Reply:
@subhtk we do not use term master and worker anymore. Please find this [guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#using-conscious-language). As I mentioned in my earlier comment, kindly figure out what master compute instances mean and replace with the required wordings.

Reply:
@jeana-redhat . Would you be OK to confirm if these `master` entries are not changeable? They seem to target engineering codebase items. As writers, we can control generic reference to components but not codebase items.

Reply:
Short answer: no, I don't think these should be master and worker in this PR. Long answer coming in a bit, want to confirm a term with dev, please stand by